docs/source/architecture/rewards_and_service_fees.rst

```diff
@@ -1,5 +1,111 @@
+:math:`D_{min} = 4` (measured in periods, with 1 period equals 7 days),
+but may be any greater number of periods.
+
+The NuCypher protocol allows service-providers to partition their stake into
+sub-stakes, up to a maximum of 30.
+A sub-stake has a unique remaining duration :math:`D` – the number of periods until
+it unlocks and the tokens become freely withdrawable. The sub-stake size :math:`\ell`
+is the number of tokens locked. It is possible to extend (but not reduce)
+:math:`D` for a sub-stake, or split a sub-stake by extending :math:`D` for part of it.
+It is also possible to acquire more tokens and increase :math:`\ell` in any sub-stake.
+This mechanism enables more granular planning for future liquidity needs.
+See :ref:`stake-management` for a more complete guide of the operations that
+can be performed over sub-stakes.
+
+Subsidy coefficient
+-------------------
+
+The amount of rewards earned by each sub-stake :math:`i` is proportional to
+the duration of time until it unlocks (:math:`D_i = t_i − t`),
+and the number of tokens in the sub-stake :math:`\ell_i`. This proportion is represented
+by a `subsidy coefficient` (:math:`\kappa`, or `kappa`) which ranges from 0.5 to 1.
+Sub-stakes that unlock in 52 periods (approximately 1 year) or more
+receive the maximum subsidy (:math:`\kappa = 1`), whereas a sub-stake that unlocks in
+4 weeks would receive slightly over half the maximum subsidy (:math:`\kappa \approx 0.54`).
+The subsidy coefficient :math:`\kappa` for a given sub-stake :math:`i` is calculated
+as following, where :math:`D_{max}` is currently set to 52 periods (i.e., 364 days):
+
+.. math::
+    \kappa_i = 0.5 \cdot \left(1 + \frac{\mathsf{min}(D_i, D_{max})}{D_{max}} \right)
+
+
+To improve user experience, instead of this coefficient,
+in NuCypher's user interfaces we display an equivalent representation called
+"`boost`", a value ranging between 1 and 2,
+and defined as :math:`\text{boost} = \frac{\kappa}{0.5}`.
+
+
+.. code:: bash
+
+    ╒═══════╤══════════╤═════════════╤═════════════╤═══════════════╤════════╤═══════════╕
+    │  Slot │ Value    │   Remaining │ Enactment   │ Termination   │  Boost │ Status    │
+    ╞═══════╪══════════╪═════════════╪═════════════╪═══════════════╪════════╪═══════════╡
+    │     0 │ 45000 NU │           5 │ Mar 24 2021 │ Apr 21 2021   │  1.10x │ DIVISIBLE │
+    ╘═══════╧══════════╧═════════════╧═════════════╧═══════════════╧════════╧═══════════╛
+
+
+Subsidies calculation
+---------------------
+
+For a given period, a sub-stake :math:`i` of size :math:`\ell_i` will receive
+the following subsidy :math:`s_i`, where :math:`L` is the sum of all locked
+sub-stakes (for all active stakers in the network) and :math:`I_{max}`
+is the maximum number of tokens that can be minted per period:
+
+.. math::
+    s_i = \kappa_i \cdot \frac{\ell_i}{L} \cdot I_{max}
+
+Note that current value for :math:`I_{max}` is 7,017,566.35 NU.
```

Comment:
The ~7M is a max and not guaranteed every period, correct? So really `s_i <= [the calculation using I_max]`...?

Reply:
No, s_i is exactly the value produced by that formula. I think what you mean is that, due to the kappa factor not being 1 for all substakes, the sum of all s_i in the network is <= I_max. Is that what you mean?

Reply:
👍 - I had it backwards